Client/src/components/Shared/Navbar/Navbar.jsx

```
@@ -1,135 +1,158 @@
+                <span className="absolute inset-0 w-full h-full bg-gradient-to-r from-blue-500 to-blue-700 rounded-full shadow-md"></span>
+                <span className="absolute inset-0 w-full h-full bg-white/15 rounded-full blur-[2px]"></span>
+                <span className="absolute inset-0 w-full h-full bg-blue-600 rounded-full transform transition-transform group-hover:scale-105"></span>
+                <span className="relative">Login</span>
+              </Link>
             )}
             
-            {/* Mobile Menu Toggle */}
-            <Navbar.Toggle className="ml-2" />
+            {/* Mobile menu button */}
+            <button className="md:hidden text-gray-300 hover:text-white focus:outline-none">
+              <svg className="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)